### DISCUSSION TITLE
How can I set sort and comp NLS parameters in connection options?

### DISCUSSION THREAD

**Discussion by paulacgates** [sijms/go-ora#436] [Q&A]
I've been trying to set sort and comp in my connection options like this: 
```
urlOptions = map[string]string{
			"language":  "BRAZILIAN PORTUGUESE",
			"territory": "BRAZIL",
			"sort":      "BINARY_AI",
			"comp":      "LINGUISTIC",
			"LOB FETCH": "POST",
		}
```

but with no success in my queries

**Comment by sijms** [comment]
connection options sort and comp is not supported
but you can change any nls parameter with alter session

**Reply by paulacgates** [reply]
Thank you for answering, I'll do that! Could this be a feature in the next releases? I've seen the type NLSData struct with "sort" and "comp", that's why I thought it worked.
